EventStore.Client.sln.DotSettings

```diff
@@ -0,0 +1,405 @@
+﻿<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
```

Comment:
dotsettings and editorconfig are going to conflict, aren't they?

Reply:
I believe they will and we can create a new PR to align those.